src/main/scala/analysis/ContextTransfer.scala

```diff
@@ -0,0 +1,75 @@
+package analysis
+import ir.*
+import util.Logger
+
+import scala.collection.mutable
+
+/** Abandoned helper analysis for intraproc steensgaard to transfer the context of the constant
+ *  propagation analysis to the call sites.
+  */
```

Comment:
If this is abandoned, do we need to keep it around?

Reply:
@yousifpatti

Reply:
It can be removed. I will remove it.

Reply:
@yousifpatti What needs to happen here, just the whole file is fine to remove since we don't need it?